### ISSUE TITLE
Move to `gulp-htmlmin` from `gulp-minify-html`

### ISSUE THREAD

**Issue by agentmilindu** [scorelab/Bassa#252]
```
npm WARN deprecated gulp-minify-html@0.1.8: Please use gulp-htmlmin
```

**Comment by tanmayanand44** [comment]
@agentmilindu I would like to work on this issue.

**Comment by agentmilindu** [comment]
@tanmayanand44 Awesome!

**Comment by tanmayanand44** [comment]
@agentmilindu Can u guide me a little.

**Comment by agentmilindu** [comment]
@tanmayanand44 yeah, sure, check what `gulp-minify-html` did and `gulp-htmlmin` can do. Then change the code and see if any further changes are needed.

**Comment by tanmayanand44** [comment]
@agentmilindu Do we also need to update yarn.lock and package.lock?

**Comment by agentmilindu** [comment]
@tanmayanand44 yes, if they got updated, then commit them also.

**Comment by tanmayanand44** [comment]
@agentmilindu Sir,but they automatically install htmlmin dependency when running yarn command.

**Comment by tanmayanand44** [comment]
@agentmilindu Please review.

**Comment by vivonk** [comment]
@agentmilindu @rehrumesh can we close this? it's resolved :)
